starter-code/basic-algorithms.js

```diff
@@ -5,3 +5,51 @@
 
 
 // Lorem ipsum generator
+
+
+var hacker1 = 'Maggie';
+console.log("The driver's name is " + hacker1); 
+var hacker2 = 'Jeremie';
+console.log("The navigator's name is " + hacker2);
+
+length1 = hacker1.length;
+length2 = hacker2.length;
+
+
+if (length1 > length2) {
+ console.log("The Driver has the longest name, it has " + length1 + "characters");
+} else if (length1 < length2) {
+ console.log("Yo, navigator got the longest name, it has " + length2+ "characters");
+} else  { 
+ console.log("wow, you both got equally long names" + length1 + "  characters!!");
+}
+var name = "";
+for (var i = 0 ; i < length1 ; i++){
+  console.log(hacker1[i].toUpperCase());
+  name += hacker1[i].toUpperCase() + " ";
+}
+console.log(name);
+var hacker2 = 'Jeremie';
+length2 = hacker2.length;
+var reversename = "";
+for (var i = length2 - 1 ; i >= 0 ; i--){
+  console.log(hacker2[i]);
+  reversename += hacker2[i] ;
+}
+console.log(reversename);
+
+if(hacker1[0] === hacker2[0] ){
```

Comment:
In your case the driver name is always first, you should try this method to compare two string (.localeCompare())
This will return : 
-     -1 if the reference string is sorted before the compareString
-     0 if the two strings are equal
-     1 if the reference string is sorted after the compareString

You can use it like this in your code : 

var n = hacker1.localeCompare(hacker2)

if (n < 0) { console.log("The driver's  name goes first")}
else if (n > 0){console.log("The navigator's name goes first")}
else {console.log("Wow, you both have the same name!")}